cloud/scope/clients.go

```diff
@@ -40,6 +42,8 @@ func (c *AzureClients) setCredentials(subscriptionID string) error {
 	if err != nil {
 		return err
 	}
+	c.ResourceManagerEndpoint = settings.Environment.ResourceManagerEndpoint
+	c.ResourceManagerVMDNSSuffix = settings.Environment.ResourceManagerVMDNSSuffix
```

Comment:
@devigned I'm not sure about this, in the sdk `GetSettingsFromFile` it doesn't return `ResourceManagerVMDNSSuffix` but its used by the cluster reconciler and it has different values based on the cloud used, is there somewhere else to get it?